app/policies/licenses/v1x5/user_policy.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Licenses::V1x5
+  class UserPolicy < ApplicationPolicy
+    authorize :license
+
+    def show?
+      verify_permissions!('user.read')
+      verify_environment!(
+        strict: false,
+      )
+
+      case bearer
+      in role: Role(:admin | :developer | :sales_agent | :support_agent | :read_only | :environment)
+        allow!
+      in role: Role(:product) if license.product == bearer
+        allow!
+      in role: Role(:user) if license.owner == bearer
+        allow!
+      in role: Role(:license) if license == bearer
+        allow!
+      else
+        deny!
+      end
+    end
+
+    def update?
+      verify_permissions!('license.user.update')
```

Comment:
Just a note to future self: this is backwards compatible with the new `license.owner.update` permission, since requests to `PUT /licenses/<license>/user` use this policy and not the new owner policy.